.github/workflows/ci.yml

```diff
@@ -38,11 +38,37 @@ jobs:
         with:
           command: test
           args: --verbose --all
+
+  ignored-test:
+    name: Large Tests (+${{ matrix.rust }}) on ${{ matrix.os }}
+    # The large timeout is to accommodate Windows builds
+    timeout-minutes: 45
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        rust: [stable, beta]
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - name: cargo fetch
+        uses: actions-rs/cargo@v1
+        with:
+          command: fetch
+      - name: Install LLVM on Windows
+        if: matrix.os == 'windows-latest'
+        run: choco install llvm -y
+      - name: Skip network tests on Ubuntu and Windows
+        # Ubuntu runners don't have network or DNS configured during test steps
+        # Windows runners have an unreliable network
+        if: matrix.os == 'ubuntu-latest' || matrix.os == 'windows-latest'
+        run: echo "ZEBRA_SKIP_NETWORK_TESTS=1" >> $GITHUB_ENV
```

Comment:
This comment is obviously not true, because some of our network tests work on Windows and Ubuntu.

Let's open a follow-up ticket to re-enable these tests with high priority?
(Because skipping sync tests on multiple platforms is risky. And we should be much more stable now.)